auth-service/src/main/kotlin/uk/gov/communities/delta/auth/plugins/Routing.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.communities.delta.auth.plugins
+
+import io.ktor.server.routing.*
+import io.ktor.server.response.*
+import io.ktor.server.application.*
+
+fun Application.configureRouting() {
```

Comment:
How come there was no need to `install(Routing)`? From my very brief look at ktor docs it says to do that.

Reply:
Looks like have a `routing {...}` block does it for you

https://ktor.io/docs/routing-in-ktor.html
> there is a convenient `routing` function that makes it simpler to install routing
> [...] `install(Routing)` is replaced with the `routing` function